http-clients/netty-nio-client/src/main/java/software/amazon/awssdk/http/nio/netty/internal/NettyRequestExecutor.java

```diff
@@ -130,10 +128,7 @@ public NettyRequestExecutor(RequestContext context) {
                         }
                     });
                 } catch (Throwable exc) {
-                    ChannelLogHelper.warn(log,
-                                          ch,
-                                          () -> "Unable to add a task to cancel the request to channel's EventLoop",
-                                          exc);
+                    log.warn(() -> "Unable to add a task to cancel the request to channel's EventLoop", exc);
```

Comment:
We seem to have lost the `ch` parameter here.

Reply:
Wondering how we can make this less error-prone in the future. Since it's an internal API, should we require an explicit, null `Channel` parameter?

Reply:
yeah I went back and forth on this; I'm going to err on caution and just remove the override that passes in `null`, and force the caller to do that.